proto/einride/example/freight/v1/freight_service.proto

```diff
@@ -171,10 +173,7 @@ message UpdateShipperRequest {
   Shipper shipper = 1 [(google.api.field_behavior) = REQUIRED];
 
   // The list of fields to be updated.
-  google.protobuf.FieldMask update_mask = 2;
-
-  // The current etag of the shipper.
-  string etag = 3 [(google.api.field_behavior) = OPTIONAL];
```

Comment:
When the resource has a etag field, that should be set on the resource rather than the update request object, see
https://google.aip.dev/154#etags-on-request-methods

Reply:
Ah yes of course!

Reply:
I think this has changed, was surprised as well.
I see that we need to update the generated test as well, to look for etag on the resource it self.

Reply:
Nice find!